test/ranking.jl

```diff
@@ -37,3 +37,31 @@ s = ["c", "a", "b", "d", "d", "b", "e", "d"] # s is a vector of strings ordered
 @test tiedrank(s) == tiedrank(x)
 @test tiedrank(x, rev = true) == tiedrank(-x)
 @test tiedrank(x, lt = (x, y) -> isless(y, x)) == tiedrank(-x)
+
+@testset "quantilerank and percentilerank" begin
+
+    pcrk = percentilerank
+    qtrk = quantilerank
+ 
+    @testset "score as number and array" begin        
+        for (method, res1, res2) in [(:rank, 40.0, [40.0, 85.0]), 
+                                    (:strict, 30.0, [30.0, 80.0]),
+                                    (:weak, 50.0, [50.0, 90.0]), 
+                                    (:mean, 40.0, [40.0, 85.0])]
+            @test pcrk([1,2,3,4,4,5,6,7,8,9], 4, method=method) == res1
+            @test qtrk([1,2,3,4,4,5,6,7,8,9], 4, method=method) == res1 / 100
+            @test pcrk([1,2,3,4,4,5,6,7,8,9], [4, 8], method=method) == res2
+            @test qtrk([1,2,3,4,4,5,6,7,8,9], [4, 8], method=method) == res2 / 100
+        end
+    end
+ 
+    @testset "missing and error" begin
+        for method in (:rank, :mean, :strict, :weak)
+            @test pcrk([1,2,3,5,6,7,8,9,10,11], 4, method=method) == 30
+            @test qtrk([1,2,3,5,6,7,8,9,10,11], 4, method=method) == 0.3
+        end
```

Comment:
What does this test? Not missing nor error AFAICT.

Reply:
Done at commit 11671f4.